build/media-suite_helper.sh

```diff
@@ -1083,24 +1083,36 @@ do_custom_patches() {
 do_cmake() {
     local bindir=""
     local root=".."
-    case "$1" in
-    global|audio|video)
-        bindir="-DCMAKE_INSTALL_BINDIR=$LOCALDESTDIR/bin-$1" ;;
-    *)
-        [[ -d "./$1" ]] && root="../$1" || bindir="$1" ;;
-    esac
-    shift 1
+    local cmakebuilddir=""
+    while (( "$#" )); do
+        case "$1" in
+        global|audio|video)
+            bindir="-DCMAKE_INSTALL_BINDIR=$LOCALDESTDIR/bin-$1"; shift ;;
+        builddir=*)
+            cmake_build_dir="${1#*=}"; shift ;;
+        skip_build_dir)
+            local skip_build_dir=y; shift ;;
+        *)
+            if [[ -d "./$1" ]]; then
+                [[ -n "$skip_build_dir" ]] && root="./$1" || root="../$1"
+            else
+                bindir="$1"
+            fi
+            shift && break ;;
+        esac
+    done
 
     local PKG_CONFIG="$LOCALDESTDIR/bin/ab-pkg-config-static.bat"
-    create_build_dir
+    [[ -z "$skip_build_dir" ]] && create_build_dir $cmake_build_dir
```

Comment:
Shouldn't `create_build_dir` always run, though? Or is this specifically for in-root builds?

Reply:
This is for the builds where a build for is already created, just not with `build*_-64-bits`

Reply:
Oh, right, sure.